<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NicknameDialog</class>
 <widget class="QDialog" name="NicknameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: '#0d3d4c';
}</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>15</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 30pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Multi Chat</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lineLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>70</y>
     <width>201</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: '#101014';
	border-radius: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 4px solid '#101014';
	font: 700 17pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1.4px;
	color: '#e3e3e3';
}

QPushButton::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneBtn">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 4px solid '#101014';
	font: 700 17pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1.4px;
	color: '#e3e3e3';
}

QPushButton::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nickBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 12pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Nickname</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
